### PR TITLE
add custom calculation method and tune timings

### PR DESCRIPTION
**Pull request by dheche**
I would like to add custom calculation method and tune timings.

### CODE REVIEW

File: prayer_times_calculator/pray_times_calculator.py [hunk 1]
```diff
@@ -38,6 +39,19 @@ def __init__(
...
+        imsak_tune = 0,
+        fajr_tune = 0,
+        sunrise_tune = 0,
+        dhuhr_tune = 0,
+        asr_tune = 0,
+        maghrib_tune = 0,
+        sunset_tune = 0,
+        isha_tune = 0,
+        midnight_tune = 0,
+        fajr_angle = "15",
```
Comment (uchagani) [inline prayer_times_calculator/pray_times_calculator.py]: should we set the defaults to 15 here?

File: prayer_times_calculator/pray_times_calculator.py [hunk 2]
```diff
@@ -72,10 +86,29 @@ def __init__(
...
+        else: self._method_settings = False
 
         date_parsed = datetime.strptime(date, "%Y-%m-%d")
         self._timestamp = int(date_parsed.timestamp())
 
+    def custom_method(self, fajr_angle, maghrib_angle, isha_angle):
+            if fajr_angle is None: fajr_angle = "null"
+            if maghrib_angle is None: maghrib_angle = "null"
+            if isha_angle is None: isha_angle = "null"
+            self._method_settings = str(fajr_angle) + ',' + str(maghrib_angle) \
```
Comment (uchagani) [inline prayer_times_calculator/pray_times_calculator.py:109]: you're setting all the angles to string already so I don't think we need to convert to string here, correct?

**Review by uchagani — CHANGES_REQUESTED** [review]

File: prayer_times_calculator/pray_times_calculator.py (hunk 2 above)
Comment (dheche) [reply prayer_times_calculator/pray_times_calculator.py:109]: nope. they raised error when i try to ommit the conversion. 
```python 
TypeError: unsupported operand type(s) for +: 'int' and 'str' 
```

File: prayer_times_calculator/pray_times_calculator.py (hunk 1 above)
Comment (dheche) [reply prayer_times_calculator/pray_times_calculator.py]: actually that is not necessary to set the default to 15 here, i set it to 15 there just to make it clear that aladhan api will use 15 as default even when we not set it.

Comment (dheche) [reply prayer_times_calculator/pray_times_calculator.py]: or maybe if you prefer to leave it to ommit the default  value, i will change it.

**Comment by uchagani** [comment]
Yes please omit it so the default is left up to the API.

File: prayer_times_calculator/pray_times_calculator.py (hunk 1 above)
Comment (dheche) [reply prayer_times_calculator/pray_times_calculator.py]: done.

**Review by uchagani — APPROVED** [review]
LGTM.  thank you.

**Pull request merged**